src/main/java/com/crowdin/util/PlaceholderUtil.java

```diff
@@ -0,0 +1,44 @@
+package com.crowdin.util;
+
+import com.crowdin.client.languages.model.Language;
+import lombok.NonNull;
+import org.apache.commons.io.FilenameUtils;
+
+public class PlaceholderUtil {
```

Comment:
I assume we have the same class in CLI project. Maybe we can later on put it to java client library and stop producing code duplicate.

Reply:
Not sure that it should be in API client :)

Reply:
still I think that this is something which might be useful in many places. But yep, client is just an integration with REST API and nothing more. Maybe we will came up later on with java sdk which will keep all those "extra" things :)